### PR TITLE
Feature/ids for UI tests

### PR DESCRIPTION
**Pull request by SteveHamann**
New IDs for elements frequently used in UI testing:
- Logout button
- New {MMC_OBJECT} buttons (e.g., 'New Client')
- Info icons in the home menu
- Dropdown menu for the anonymous group

### CODE REVIEW

File: frontend/src/components/Info.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ const useStyles = makeStyles((theme) => ({
 	}
 }));
 
-const Info = ({className, infos, infoIcon, infoTooltip, infoLink, actionIcon, label, chart, icon, alignment, ...rest}) => {
+const Info = ({className, infos, infoIcon, infoTooltip, infoLink, actionIcon, label, chart, icon, alignment, infoId, ...rest}) => {
```
Comment (anreinha) [inline frontend/src/components/Info.js:40]: Rename `infoId` to just `id`.

File: frontend/src/components/Status.js [hunk 2]
```diff
@@ -308,6 +308,7 @@ const Status = ({
 								<Info
 									label="Broker Traffic"
 									infoIcon
+									infoId={"broker-traffic-info-icon"}
```
Comment (anreinha) [inline frontend/src/components/Status.js:311]: Rename `infoId` to just `id`. (also the following occurrences.

**Review by anreinha — CHANGES_REQUESTED** [review]

File: frontend/src/components/Info.js (hunk 1 above)
Comment (SteveHamann) [reply frontend/src/components/Info.js]: Done, see latest commit.

File: frontend/src/components/Status.js (hunk 2 above)
Comment (SteveHamann) [reply frontend/src/components/Status.js]: Done, see latest commit.

**Pull request merged**